services/core/MasterDriverAgent/master_driver/interfaces/modbus_tk/server.py

```diff
@@ -181,8 +181,8 @@ def define_slave(self, slave_id, client_class, unsigned=True):
             raise TypeError("client_class must be subclass of {0}".format(Client))
 
         for request in client_class().requests():
-            logger.info("Adding [%s], %s, %s, %s", request.name, request.table, request.address, request.count)
-            slave.add_block(request.name, request.table, request.address, request.count)
+            logger.info("Adding %s", request)
+            slave.add_block(*request.block_info())
```

Comment:
I think you should go back to the previous way of doing this for the reasons mentioned above.

Reply:
I fixed it.